### PR TITLE
new sequence length section added

### PR DESCRIPTION
**Pull request by moonlanderr**
\<insert pull request description here\>

-----

# Checklist

Please go through each entry in the below checklist and mark an 'X' if that condition has been met. Every entry should be marked with an 'X' to be get the Pull Request approved.

- [ ] All `import`s are in the first cell? First block of imports are standard libraries, second block are 3rd party libraries, third block are all `arcgis` imports? Note that in some cases, for samples, it is a good idea to keep the imports next to where they are used, particularly for uncommonly used features that we want to highlight.
- [ ] All `GIS` object instantiations are one of the following?
    - `gis = GIS()`
    - `gis = GIS('home')`
    - `gis = GIS(profile="your_online_portal")`
    - `gis = GIS('https://pythonapi.playground.esri.com/portal')`
    - `gis = GIS(profile="your_enterprise_portal")`
- [ ] If this notebook requires setup or teardown, did you add the appropriate code to `./misc/setup.py` and/or `./misc/teardown.py`?
- [ ] If this notebook references any portal items that need to be staged on AGOL/Python API playground, did you coordinate with a Python API team member to stage the item the correct way with the api\_data\_owner user?
- [ ] Code refactored & split out across multiple cells, useful comments?
- [ ] Consistent voice/tense/narrative style? Thoroughly checked for typos?
- [ ] All images used like `` instead of ``? All map widgets contain a static image preview? (Call `mapview_inst.take_screenshot()` to do so)
- [ ] All file paths are constructed in an OS-agnostic fashion with `os.path.join()`? (Instead of `r"\foo\bar"`, `os.path.join(os.path.sep, "foo", "bar")`, etc.)
- [ ] **IF YOU WANT THIS SAMPLE TO BE DISPLAYED ON THE DEVELOPERS.ARCGIS.COM WEBSITE**, ping @ DavidJVitale so he can add it to the list for the next deploy

### CODE REVIEW

**Comment by priyankatuteja** [comment]
fixes both https://github.com/ArcGIS/geosaurus/issues/6567 https://github.com/ArcGIS/geosaurus/issues/6287

File: guide/14-deep-learning/dl-on-time-series-data.ipynb [hunk 1]
```diff
@@ -14,14 +14,17 @@
     "# Table Of Contents\n",
```
Comment (priyankatuteja) [inline guide/14-deep-learning/dl-on-time-series-data.ipynb:14]: <p>remove line: The next step is initializing the timeseries model for model training.</p><p> </p><h1>TimeSeriesModel </h1><p>TimeSeriesModel in&nbsp;<code>arcgis.learn</code>&nbsp;is built on top of&nbsp;??&nbsp;library. The model training and inferencing workflow are similar to computer vision models in&nbsp;<code>arcgis.learn</code>.</p><p>Run the command below to see what backbones are supported for the timeseries task. </p><p> </p><pre class="ql-syntax" spellcheck="false">print(TimeSeriesModel.supported_backbones)
</pre><p> </p><p>also add imports in a call</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1033/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (priyankatuteja) [inline guide/14-deep-learning/dl-on-time-series-data.ipynb:14]: <p>the part should be moved down after model initialization</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1033/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

**Review by priyankatuteja — CHANGES_REQUESTED** [review]
I have suggested minor changes. Thanks for fixing these issues

File: guide/14-deep-learning/dl-on-time-series-data.ipynb (hunk 1 above)
Comment (moonlanderr) [reply guide/14-deep-learning/dl-on-time-series-data.ipynb:14]: <p>line removed, info on baackbones/model architecture is already there, Timeseries.supported_backbones gives error - Timeseries has no attribute supported_backbones</p>

Comment (moonlanderr) [reply guide/14-deep-learning/dl-on-time-series-data.ipynb:14]: <p>moved down after model initialization</p>

**Comment by priyankatuteja** [comment]
@moonlanderr This pr has 12 changes, i suppose there should inly be change in 1 file. please fix the pr

**Review by AtmaMani — APPROVED** [review]

**Comment by AtmaMani** [comment]
@moonlanderr I fixed the 12 images that got deleted. Changes look good thanks for the update. @priyankatuteja thanks for reviewing.

**Pull request merged**